wsf_scraping/settings.py

```diff
@@ -17,13 +17,13 @@
     'wsf_scraping.pipelines.WsfScrapingPipeline': 10,
 }
 FEED_STORAGES = {
-    'dsx': 'tools.DSXFeedStorage',
+    'aws': 'tools.AWSFeedStorage',
 }
 
 # LOG_ENABLED = False
 LOG_LEVEL = 'INFO'
 LOG_FORMATTER = 'wsf_scraping.middlewares.PoliteLogFormatter'
-LOG_FILE = 'var/log-{log_level}.txt'.format(log_level=LOG_LEVEL)
+# LOG_FILE = 'var/log-{log_level}.txt'.format(log_level=LOG_LEVEL)
```

Comment:
try not to commit removing and not removing comments. Check git-cola which is a GUI that helps you tidy up commits.